src/features/impl_alloc.rs

```diff
@@ -262,10 +278,20 @@ where
 
 impl<T> Decode for Vec<T>
 where
-    T: Decode,
+    T: Decode + 'static,
```

Comment:
Doesn't this bound rule out zero-copy deserialization of types like `Vec<&'de str>`?

Reply:
To answer my own question: No, as this case goes though `impl BorrowDecode for Vec<T>`.